compendium/modules/w06-patterns-exercise.tex

```diff
@@ -281,6 +302,49 @@
 \QUESTEND
 
 
+\WHAT{Mönstermatcha med nyttan av \code{enum}.}
```

Comment:
Remove "nyttan av" as the sentence reads a bit strange. I suggest instead the simpler title: "Mönstermatcha enumeration"